src/Controls/StreetControls.js

```diff
@@ -51,8 +51,16 @@ class StreetControls extends FirstPersonControls {
         this.player.addEventListener('animation-frame', this.updateView.bind(this));
 
         // twos positions used by this control : current and next
+        this.previousPosition = undefined;
         this.currentPosition = undefined;
         this.nextPosition = undefined;
+
+        // store key binding to go to next position, default to Z
```

Comment:
Please move those comments before the class declaration, as `@property` of the class.

Reply:
ok, done

Reply:
Thanks, merged